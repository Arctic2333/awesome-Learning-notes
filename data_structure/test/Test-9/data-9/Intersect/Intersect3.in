10
3 6 11 25 33 36 56 77 86 92
10
3 6 11 25 33 36 56 77 86 92
