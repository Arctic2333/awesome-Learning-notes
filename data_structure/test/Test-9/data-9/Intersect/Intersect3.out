3 25 218 
